curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-javascript-by-building-a-role-playing-game/62a8c4db0710f3260f867a92.md

```diff
@@ -7,17 +7,20 @@ dashedName: step-76
 
 # --description--
 
-What if the player doesn't have enough gold to buy health? When you want to run code conditionally, you can use the `if` statement. Put all of the code in your `buyHealth` function inside an `if` statement. For example:
+What if the player doesn't have enough gold to buy health? When you want to run code conditionally, you can use the <dfn>if</dfn> statement.
+
+An `if` statement is used to make decisions in code. The keyword `if` is followed by a condition in parentheses. If the condition is true, the code inside the curly braces `{}` is executed. If the condition is false, the code inside the curly braces is skipped.
+
+Here is an example of an `if` statement:
 
 ```js
-function myFunction() {
-  if ("condition") {
-    console.log("Hello World!");
-  }
+const num = 5;
+if (num >= 3) {
+  console.log("This code will run because num is greater than or equal to 3.");
 }
 ```
 
-For now, the `if` statement condition should be set to the string `condition` as a placeholder.
+Start by placing all of the code in your `buyHealth` function inside an `if` statement. For the `if` statement condition,  check if `gold` is greater than or equal to `10`.
 
 # --hints--
 
```

Comment:
@netanelben i got you buddy, try these hints i wrote for you. Also, check the contributing guides, you should find some instructions on how to delete the step number 77 using the terminal[if you're on windows, believe me, its gonna be a huge pain!]. If you get stuck, reply here I'll be more than happy to help ! Also @jdwilkin4  what do you think of the solution ?

# --hints--

Your `if` statement should check if `gold` is greater than or equal to `10`.

```js
assert.match(buyHealth.toString(), /if\s*\(\s*gold\s*>=\s*10\s*\)/);
```

All of your `buyHealth` code should be inside the `if` statement. If you've deleted the original code, you can reset the step and try again.

```js
assert.match(buyHealth.toString(), /if\s*\(\s*gold\s*>=\s*10\s*\)\s*{\s*gold\s*-=\s*10;\s*health\s*\+=\s*10;\s*goldText\.innerText\s*=\s*gold;\s*healthText\.innerText\s*=\s*health;\s*}/);
```

Reply:
thanks @JoyShaheb i am looking into that right now. I am using mac of-curse ^_^

Reply:
@netanelben cheers buddy 🥂[meanwhile I'm stuck with windows :')]

Reply:
I need some assistant here 
I'm trying to run the `delete-step` script and getting this err:
```Error: ENOENT: no such file or directory, open '/Users/nate/_meta/curriculum/meta.json'```

Reply:
Are you running the script in the correct directory?

If you are coming from the root directory, open up a new tab in the terminal.
Then run this command to get to the correct directory ` cd curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-javascript-by-building-a-role-playing-game/`

then run the delete command `pnpm run delete-step 77`

Reply:
We can remove this part here from the hint text

> If you've deleted the original code, you can reset the step and try again.